src/Http/Resources/V1/Admin/Setting/ExchangeRateResource.php

```diff
@@ -17,7 +17,7 @@ public function toArray($request)
         return [
             'id'              => $this->id,
             'rate'            => $this->rate,
-            'target_currency' => $this->target_currency,
+            'currency_name'   => $this->currency()->first()->name,
```

Comment:
The 'target_currency' key should be the same. In the backend `target_currency` is linked to the currency table. Use the resource here.

Reply:
fixed